external-crates/move/crates/move-analyzer/editors/code/scripts/create_from_local.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/zsh
+# Copyright (c) Mysten Labs, Inc.
+# Modifications Copyright (c) 2024 IOTA Stiftung
```

Comment:
Is this actually a mysten script that we are pulling?

Reply:
Yeah, but modified https://github.com/MystenLabs/sui/blob/5c06cf64cddbe537bcc992231d94b9211b067ece/external-crates/move/crates/move-analyzer/editors/code/scripts/create_from_local.sh#L4
They seem to run this locally manually and not in the CI